test/run

```diff
@@ -261,7 +229,7 @@ run_doc_test() {
   docker run --rm ${IMAGE_NAME} /bin/bash -c "cat /help.1" >${tmpdir}/help.1
   # Check whether the help.1 file includes some important information
   for term in 6379 "REDIS\_PASSWORD" volume; do
-    if ! cat ${tmpdir}/help.1 | grep -F -q -e "${term}" ; then
+    if ! cat ${tmpdir}/help.1 | grep -q -e "${term}" ; then
```

Comment:
We have already solved similar pull request here: https://github.com/sclorg/container-common-scripts/pull/249.

I guess, we should use `-E` instead of `-F`.

Reply:
Done